### ISSUE TITLE
RECORD_LAYOUT fields are not sorted correctly

### ISSUE THREAD

**Issue by matiss** [DanielT/a2lfile#29]
`RECORD_LAYOUT` fields should be sorted by the position (index) in an ascending order. Some tools give warning about this.

Current behavior:
```
/begin RECORD_LAYOUT
      MAP_XU16_YU16_ZU16
      AXIS_PTS_X 3 UWORD INDEX_INCR DIRECT
      AXIS_PTS_Y 4 UWORD INDEX_INCR DIRECT
      FNC_VALUES 5 UWORD ROW_DIR DIRECT
      NO_AXIS_PTS_X 1 UBYTE
      NO_AXIS_PTS_Y 2 UBYTE
/end RECORD_LAYOUT
```

Expected behavior:
```
/begin RECORD_LAYOUT
      MAP_XU16_YU16_ZU16
      NO_AXIS_PTS_X 1 UBYTE
      NO_AXIS_PTS_Y 2 UBYTE
      AXIS_PTS_X 3 UWORD INDEX_INCR DIRECT
      AXIS_PTS_Y 4 UWORD INDEX_INCR DIRECT
      FNC_VALUES 5 UWORD ROW_DIR DIRECT
/end RECORD_LAYOUT
```

**Comment by DanielT** [comment]
Thanks for the report.

The issue is this: there is no special case for the ordering of newly created RECORD_LAYOUTs in the code, so its optional components are written alphabetically. RECORD_LAYOUTS that were read from a file have attached layout info, so they don't have this issue.

**Comment by DanielT** [comment]
I just published a2lfile 1.5.0 with this fix and several others
